app/frontend/styles/components/check_your_answers.scss

```diff
@@ -100,6 +100,7 @@ dt.app-check-your-answers__question > h4 {
 }
 
 .change-button {
+  font-weight: normal;
```

Comment:
Have you checked this with UX? Prototype pulls rank over Figma most of the time

Reply:
No I haven't. Good to know.